compiler/src/iree/compiler/Dialect/Util/Transforms/ConvertPrimitiveType.cpp

```diff
@@ -34,6 +36,42 @@ namespace IREE {
 namespace Util {
 namespace {
 
+Value convertRankedFloat(OpBuilder &builder, Type type, ValueRange inputs,
+                         Location loc) {
+  Type eTy = getElementTypeOrSelf(type);
+  Type inputETy = getElementTypeOrSelf(inputs[0].getType());
+  if (!getElementTypeOrSelf(type).isa<FloatType>()) return nullptr;
+
+  if (inputETy.getIntOrFloatBitWidth() > eTy.getIntOrFloatBitWidth())
```

Comment:
Nit: Please add braces. (this diverges from LLVM style but is recommended in IREE)

Reply:
Fixed.